go/models.go

```diff
@@ -0,0 +1,68 @@
+package examples
```

Comment:
Should there be region tags in this file?

Reply:
Will add them.